setup.py

```diff
@@ -1,4 +1,3 @@
-import os
 import sys
 import pathlib
```

Comment:
It looks like tools_path, base_prefix and data_files_path are not needed anymore. If that's correct, please remove them and this import as well.

Reply:
Good catch! I need to check my other updates as well. And remove schema from conda/meta.yaml.